getting-started/templates/systemlink-values.yaml

```diff
@@ -84,6 +84,16 @@ global:
     ## CDN endpoint for telemetry-driven content. NI does not recommend changing this value.
     ##
     cdnEndpoint: "https://web-sdk.ni.com/"
+  ##
+  ## MongoDB Database Configuration.
+  ## Many Systemlink Enterprise services utilize MongoDB databases for data storage. Customers can deploy their own MongoDB instance 
+  ## or utilize a managed solution such as MongoDB Atlas. Once provisioned provide the connection details in global.secrets.mongodb.connection_string.
+  ##
+  mongodb:
+    ## Enable or disable installation of per-service MongoDB instances. 
+    #  <ATTENTION> - If you do *not* have your own MongoDB instance set this (`global.mongodb.install`) to `true`.
+    ##
+    install: false 
```

Comment:
@BKnight760 I left this as `false` since we really want customers to go in this direction, but we left the default in the high-level systemlink chart to `true` to keep current customers from having to scramble.

Are you ok with this or should I default this to `true` as well to match the high-level chart?

Reply:
Yes, I think install:false is the correct thing to do for now.